covid.py

```diff
@@ -6,22 +6,81 @@
 
 @st.cache(ttl=3600)
 def covid():
-    df = data.covid()
-    for int_col in ("Confirmed", "Deaths", "Recovered", "Active"):
-        df[int_col] = df[int_col].fillna(0).astype("int")
-    return df
+    return data.covid()
 
 
 @st.cache
 def population():
     return data.population()
 
 
+@st.cache(ttl=3600)
+def merge():
+    cdf = covid()
+    for int_col in ("Confirmed", "Deaths", "Recovered", "Active"):
+        cdf[int_col] = cdf[int_col].fillna(0).astype("int")
+    cdf["Country_Region"] = cdf["Country_Region"].replace(
```

Comment:
Since we decided to now have two data frames -- one for country normalized by population, and one not joined with population, we can probably put this normalization back into the original functions. I'll do a follow up PR for that.

Reply:
SGTM!